node/src/command.rs

```diff
@@ -62,26 +60,15 @@ impl<T: sc_service::ChainSpec + 'static> IdentifyChain for T {
 	}
 }
 
-fn load_spec(
-	id: &str,
-	para_id: ParaId,
-) -> std::result::Result<Box<dyn sc_service::ChainSpec>, String> {
+fn load_spec(id: &str) -> std::result::Result<Box<dyn sc_service::ChainSpec>, String> {
 	match id {
 		"centrifuge" => Ok(Box::new(chain_spec::centrifuge_config())),
-		"centrifuge-dev" => Ok(Box::new(chain_spec::centrifuge_dev(para_id))),
-		"centrifuge-local" => Ok(Box::new(chain_spec::centrifuge_local(para_id))),
+		"centrifuge-local" => Ok(Box::new(chain_spec::centrifuge_local(LOCAL_PARA_ID))),
 		"altair" => Ok(Box::new(chain_spec::altair_config())),
-		"altair-dev" => Ok(Box::new(chain_spec::altair_dev(para_id))),
-		"altair-local" => Ok(Box::new(chain_spec::altair_local(para_id))),
+		"altair-local" => Ok(Box::new(chain_spec::altair_local(LOCAL_PARA_ID))),
 		"catalyst" => Ok(Box::new(chain_spec::catalyst_config())),
-		"catalyst-local" => Ok(Box::new(chain_spec::catalyst_local(para_id))),
-		"antares" => Ok(Box::new(chain_spec::antares_config())),
-		"antares-local" => Ok(Box::new(chain_spec::antares_local(para_id))),
-		"charcoal" => Ok(Box::new(chain_spec::charcoal_config())),
-		"charcoal-local" => Ok(Box::new(chain_spec::charcoal_local(para_id))),
-		"demo" => Ok(Box::new(chain_spec::demo(para_id))),
-		"development" => Ok(Box::new(chain_spec::development(para_id))),
-		"development-local" => Ok(Box::new(chain_spec::development_local(para_id))),
```

Comment:
I think we've broken our development benchmarking with this. We didn't have a CI job for development (I do not remember why), and we didn't notice it.

We should call it `development-local` or modify `check_benchmarks.sh` script.

cc @wischli